subplays/prep_box.yaml

```diff
@@ -33,14 +33,14 @@
     retries: 5
 
   # TODO: uncomment with proper url once we have it
-  # - name: get prebuilt binaries archive from dogfood
-  # - get_url:
-  # -   url: "http://fqdn.com/files/puppet-agent-build/el{{ el_version }}-{{ el_arch }}.tar.gz"
-  # -   dest: "/el{{ el_version }}-{{ el_arch }}.tar.gz"
-  # -   validate_certs: false
-  # -   timeout: 30
-  # - register: dogfood_build_binaries
-  # - ignore_errors: true
+  - name: get prebuilt binaries archive from dogfood
+    get_url:
+      url: "http://fqdn.com/files/puppet-agent-build/el{{ el_version }}-{{ el_arch }}.tar.gz"
```

Comment:
just a placeholder. though, now that i think about it, i should probably use a variable for the web address since we don't want to check-in the address of the dogfood server.